firestore/integration_test_internal/src/firestore_test.cc

```diff
@@ -60,7 +62,14 @@ using ::firebase::auth::Auth;
 using ::testing::ContainerEq;
 using ::testing::HasSubstr;
 
-TEST_F(FirestoreIntegrationTest, GetInstance) {
+class FirestoreTest : public FirestoreIntegrationTest {
+ protected:
+  const std::string GetFirestoreDatabaseId(Firestore* firestore) {
+    return firestore->internal_->database_id().database_id();
```

Comment:
we do not have `GetInternal` static method in `Firestore` class, I can add `GetInternal` in `FirestoreTest` class, but so far, it will be only called by this `GetFirestoreDatabaseId` method.

Reply:
Imported from convertor file.